scripts/apps/vocabularies/constants.ts

```diff
@@ -15,6 +15,21 @@ export const MEDIA_TYPES = {
 
 export const MEDIA_TYPE_KEYS = Object.keys(MEDIA_TYPES).map((type) => MEDIA_TYPES[type].id);
 
+export const VOCABULARY_SELECTION_TYPES = {
+    SINGLE_SELECTION: {
+        id: 'single selection',
+        label: gettext('Single selection')
+    },
+    MULTIPLE_SELECTION: {
+        id: 'multi selection',
+        label: gettext('Multi selection')
+    },
+    DO_NOT_SHOW: {
```

Comment:
I can't find any usages of this. Is it needed?

Reply:
Yes, there is a task about it:
https://dev.sourcefabric.org/browse/SDESK-3625

Reply:
Are going to implement this in another PR or should it be working right now?

Reply:
The task is still in TODO, so the fix will be in a different PR.